app/components/quantity-widget.js

```diff
@@ -0,0 +1,50 @@
+import { computed } from "@ember/object";
+import Component from "@ember/component";
+
+export default Component.extend({
+  value: computed("model.package", function() {
+    if (this.get("package.requestedPackage")) {
+      return this.get("package.requestedPackage.quantity");
+    }
+    return 1;
+  }),
+  setValueIfValid(value, isValidValue, id) {
+    if (isValidValue) {
+      this.set("showErrorMessage", false);
+      Ember.$("#qty" + id).val(value);
+    } else {
+      this.set("showErrorMessage", true);
+    }
+  },
+  elementValue(id) {
+    return Ember.$("#qty" + id).val();
```

Comment:
same as above,

`return this.get('value')`

Reply:
u can completely get rid of `elementValue` property since you have `value` property

Reply:
as mentioned above it used to avoid 2 way binding